mlir/test/Dialect/XeGPU/ops.mlir

```diff
@@ -751,4 +751,51 @@ gpu.func @fence() {
   gpu.return
 }
 
+// CHECK-LABEL: gpu.func @create_matrix_desc({{.*}}) {
```

Comment:
Can you add a test with layout attribute?
Without layout attribute, matrix_desc is not different from a memref created by  `memref.view`
I appears to me, what sets matrix_desc apart from memref is the optional layout attribute.

Reply:
matrix_desc supports blocked layout which memref doesn't support.